tests/e2e/cucumber/features/shares/share.feature

```diff
@@ -83,15 +83,65 @@ Feature: share
       | sharedFile.txt      | txtFile    | some text |
     And "Alice" uploads the following resource
       | resource        |
-      | testavatar.jpeg |
       | simple.pdf      |
-    When "Alice" shares the following resource using the sidebar panel
+      | sampleGif.gif   |
+      | testimage.mp3   |
+      | sampleOgg.ogg   |
+      | sampleWebm.webm |
+      | test_video.mp4  |
+      | testavatar.jpeg |
+      | testavatar.png  |
+    When "Alice" views the file "testavatar.png" in the preview app using the sidebar panel
+    Then "Alice" is in a media-viewer
+    When "Alice" closes the file viewer
+    And "Alice" opens the following file in mediaviewer
+      | resource        |
+      | testavatar.jpeg |
+    Then "Alice" is in a media-viewer
+    When "Alice" navigates to the next media resource
+    When "Alice" navigates to the previous media resource
+    When "Alice" closes the file viewer
+    When "Alice" opens the following file in mediaviewer
+      | resource        |
+      | sampleGif.gif   |
+    Then "Alice" is in a media-viewer
+    When "Alice" closes the file viewer
+    And "Alice" opens the following file in mediaviewer
+      | resource      |
+      | testimage.mp3 |
+    Then "Alice" is in a media-viewer
+    When "Alice" closes the file viewer
+    And "Alice" opens the following file in mediaviewer
+      | resource      |
+      | sampleOgg.ogg |
+    Then "Alice" is in a media-viewer
+    When "Alice" closes the file viewer
+    And "Alice" opens the following file in mediaviewer
+      | resource        |
+      | sampleWebm.webm |
+    Then "Alice" is in a media-viewer
+    When "Alice" closes the file viewer
+    And "Alice" opens the following file in mediaviewer
+      | resource       |
+      | test_video.mp4 |
+    Then "Alice" is in a media-viewer
+    When "Alice" downloads the following resource using the preview topbar
+      | resource       | type |
+      | test_video.mp4 | file |
```

Comment:
since we can download only the current opened file from preview topbar, let's not use the data table because it suggests that we can add more files/folders
```suggestion
    When "Alice" downloads a file "test_video.mp4" using the preview topbar
```

Reply:
I tried to make use of the same step implementation. Do I have to create new step?

Reply:
I see. should be fine